proto/directions_options.proto

```diff
@@ -19,6 +19,12 @@ enum FilterAction {
   include = 1;
 }
 
+enum DirectionsType {
```

Comment:
the naming is kind of crappy here, any suggestions?

Reply:
@kevinkreiser i gave it serious thought and could think of anything better